### ISSUE TITLE
We need a way to refer to the code instructions in the preface

### ISSUE THREAD

**Issue by AllenDowney** [AllenDowney/ThinkJava#4]
There is a section in the preface about how to work with the code.  We want to refer to it in the exercise section at the end of the book.  The current reference is not working, but we don't know why.

**Comment by DanGitOreilly** [comment]
The problem here is a missing `id` in the `sect1` to which you wish to point.

Your xref markup looks fine:

`See <xref linkend="code" /> for instructions on how to download the repository.`

But I could not find a matching `id="code"`

When I looked at the section that I believe you want to reference, I found a different `id`. If you change it to `id="code"`, it should work as expected.

```
<sect1 id="a0000000122" remap="section">
  <title>Using the code examples</title><para>Most of the code examples in this book are available from a Git repository at <ulink url="https://github.com/AllenDowney/ThinkJavaCode"></ulink>. Git is a “version control system” that allows you to keep track of the files that make up a project. A collection of files under Git’s control is called a “repository”. </para>
```

**Comment by ChrisMayfield** [comment]
@AllenDowney Could this definition of code be what plastex is confusing with \begin{code}?

**Comment by AllenDowney** [comment]
Resolved in ae39398
